packages/react/src/AnchoredOverlay/AnchoredOverlay.tsx

```diff
@@ -198,6 +198,7 @@ export const AnchoredOverlay: React.FC<React.PropsWithChildren<AnchoredOverlayPr
           left={position?.left || 0}
           anchorSide={position?.anchorSide}
           className={className}
+          reflow={false}
```

Comment:
Added `reflow={false}` so this change doesn't affect components that consume `AnchoredOverlay` (e.g. `ActionMenu`). 

Not sure if this is ideal or not, but reduces the surface area of unexpected style changes.

Reply:
I'd say we can leave it like this here for now as it won't 'cause any regressions. Depending on what you decide to do [here](https://github.com/primer/react/pull/5129/files#r1819869013) I can port over as part of https://github.com/github/primer/issues/3420

Reply:
Definitely, I think we can do something similar in `AnchoredOverlay` to restrict it to just the component itself and not anything that's using it 🤔